### ISSUE TITLE
Album art is not resized in the web player 'poster' box

### ISSUE THREAD

**Issue by gregoo** [ampache/ampache#706]
While the catalog properly displays thumbnails when using the `resize_images` configuration, the parameter does not seem to be used by the webplayer.
When you pull out the URL image, you can see it is missing a `thumb` parameter.

I do apologize for not providing a proper pull request as I do not know if the patch I made would cause issues somewhere else. The file I edited does not really document the purpose of the function or its logic.

```diff
diff --git a/lib/class/webplayer.class.php b/lib/class/webplayer.class.php
index 4f0ee3d..df9057e 100644
--- a/lib/class/webplayer.class.php
+++ b/lib/class/webplayer.class.php
@@ -259,7 +259,7 @@ class WebPlayer
         $js['filetype'] = $types['player'];
         $js['url'] = $url;
         if ($urlinfo['type'] == 'song') {
-            $js['poster'] = $item->image_url . (!AmpConfig::get('ajax_load') ? '&thumb=4' : '');
+            $js['poster'] = $item->image_url . (!AmpConfig::get('ajax_load') ? '&thumb=4' : (AmpConfig::get('resize_images') ? '&thumb=1' : ''));
         }

         debug_event("webplayer.class.php", "Return get_media_js_param {".json_encode($js)."}", 5);
```

Basically, in the `webplayer.class.php`, in the `get_media_js_param` function, I'm adding an extra check to see if `resize_images` has been enabled and add `thumb=1` to the poster URL.

**Comment by Afterster** [comment]
Indeed, thanks for pointing this out.
Basically you identified the fix correctly. You just don't have to check for resize_images, it will be handled at thumb creation, and the thumb type should be 3 (80x80px) not 1 (100x100px).
